README.md

```diff
@@ -48,6 +48,8 @@ The following environment variables are used to launch the Red Hat AppStudio ins
 |---|---|---|---|
 | `GITHUB_TOKEN` | yes | A github token used to create AppStudio applications in github  | ''  |
 | `QUAY_TOKEN` | yes | A quay token to push components images to quay.io. Note the quay token must be your dockerconfigjson encoded in base64 format. Example: `export QUAY_TOKEN=$(base64 < ~/.docker/config.json)` | '' |
+| `DEFAULT_QUAY_ORG` | yes | A quay organization where repositories for component images will be created  | 'redhat-appstudio-qe'  |
+| `DEFAULT_QUAY_ORG_TOKEN` | yes | A quay token of OAuth application for `DEFAULT_QUAY_ORG` with scopes -  Administer organizations, Adminster repositories, Create Repositories | ''  |
```

Comment:
since these env vars are marked as required, does it make sense to also add them [here](https://github.com/redhat-appstudio/e2e-tests/blob/9b529215f76e0ef460ff96af6ee8a5bcb05cdccc/magefiles/magefile.go#L222)